File: app/src/main/java/com/karla/learningverbs/view/profile/CreateAccountActivity.java
package com.karla.learningverbs.view.profile;

import androidx.annotation.NonNull;
import androidx.lifecycle.ViewModelProvider;

import android.content.Intent;
import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;

import com.karla.learningverbs.utils.base.BaseActivity;
import com.karla.learningverbs.utils.base.BaseViewModel;

public class CreateAccountActivity extends BaseActivity<ActivityCreateAccountBinding, BaseViewModel> {

    @Override
    protected BaseViewModel createViewModel() {
        return new ViewModelProvider(this).get(BaseViewModel.class);
    }

    @NonNull
    @Override
    protected ActivityCreateAccountBinding createViewBinding(LayoutInflater layoutInflater) {
        return ActivityCreateAccountBinding.inflate(layoutInflater);
    }

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        binding.btnLogin.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                startActivity(new Intent(CreateAccountActivity.this, LoginActivity.class));

            }
        });

        binding.btnCreateAcount.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                startActivity(new Intent(CreateAccountActivity.this, SignUpActivity.class));

            }
        });

    }
}

File: app/src/main/java/com/karla/learningverbs/view/profile/CreateAccountActivity.kt
package com.karla.learningverbs.view.profile

import android.content.Intent
import android.os.Bundle
import android.view.LayoutInflater
import androidx.lifecycle.ViewModelProvider
import com.karla.learningverbs.databinding.ActivityCreateAccountBinding
import com.karla.learningverbs.utils.base.BaseActivity
import com.karla.learningverbs.utils.base.BaseViewModel

class CreateAccountActivity : BaseActivity<ActivityCreateAccountBinding?, BaseViewModel?>() {
    override fun createViewModel(): BaseViewModel {
        return ViewModelProvider(this).get(BaseViewModel::class.java)
    }

    override fun createViewBinding(layoutInflater: LayoutInflater): ActivityCreateAccountBinding {
        return ActivityCreateAccountBinding.inflate(layoutInflater)
    }

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding!!.btnLogin.setOnClickListener {
            startActivity(
                Intent(
                    this@CreateAccountActivity,
                    LoginActivity::class.java
                )
            )
        }
        binding!!.btnCreateAcount.setOnClickListener {
            startActivity(
                Intent(
                    this@CreateAccountActivity,
                    SignUpActivity::class.java
                )
            )
        }
    }
}